site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -10,7 +10,7 @@ endif::[]
 [field]#refreshToken# [type]#[String]#::
 The refresh token that can be used to obtain a new access token once the provided one has expired.
 +
-Because a refresh token is per user and per application, this value will only be returned when an [field]#applicationId# was provided on the login request.
+Because a refresh token is per user and per application, this value will only be returned when an [field]#applicationId# was provided on the login request and the user is registered to the application.
```

Comment:
That is correct. If you are not registered, you are not authorized, and thus will not be provided a refresh token. 

You can see the business logic in `BaseLoginAction.addTokensToResponse`.